pages/maps.tsx

```diff
@@ -53,6 +58,8 @@ const MapsGeneratorPage = () => {
   const [count, setCount] = useState(getInitialCount());
   const [editing, setEditing] = useState(false);
   const [copied, setCopied] = useState<null | "URL" | "LIST">(null);
+  const [displayType, setDisplayType] = useState<"TABLE" | "LIST">("LIST");
```

Comment:
I think we can just do off with the select and use Table when not mobile and the Original for mobile. With current solution there is this when resizing window:

![image](https://user-images.githubusercontent.com/38327916/128647058-c02d4b7f-e1c0-46f8-8de8-91d51af1d17e.png)

+ I think the new one is more illustrative anyway when the screen size allows it so allowing switching is not required.